### ISSUE TITLE
Add metadata to website (React-Helmet)

### ISSUE THREAD

**Issue by emielvanrijn** [svsticky/static-sticky#61]

**Comment by JustNireon** [comment]
This ticket is 5yo, I'm not quite sure if this is still relevant or what was meant with this originally. Would just close it unless people know what metadata should be added

**Comment by josvanos** [comment]
@SilasPeters I think this issue suggests to add SEO fields such as <meta name="description" content="..." />. The head of static sticky shows no seo fields are present.

**Comment by SilasPeters** [comment]
> @SilasPeters I think this issue suggests to add SEO fields such as <meta name="description" content="..." />. The head of static sticky shows no seo fields are present. 

I will just mark this as a "good first issue", as ``<meta name="description" content="..." />.`` is quite neat to have. Any objections?

**Comment by josvanos** [comment]
Yeah! if the description is on point, I think ranking in Google would improve and we would be easier found.

Meta robots is also missing.

**Comment by SilasPeters** [comment]
I updated the description. Thanks guys!
